samples/sensor/vl53l0x/prj.conf

```diff
@@ -0,0 +1,12 @@
+CONFIG_I2C=y
+CONFIG_GPIO=y
+CONFIG_SENSOR=y
+CONFIG_VL53L0X=y
+CONFIG_VL53L0X_PROXIMITY_THRESHOLD=100
+
+CONFIG_NEWLIB_LIBC=y
```

Comment:
I agree with paul, lets add a patch in here, to introduce abs into lib/libc/minimal/include/stdlib.h

something like:

#define abs(x) ((x)<0 ? -(x) : (x))

This removes the NEWLIB dependency.